content/millennium/r4/clinical/care-provision/nutrition-order.md

```diff
@@ -16,6 +16,7 @@ The following fields are returned if valued:
 * [NutritionOrder id](https://hl7.org/fhir/r4/resource-definitions.html#Resource.id)
 * [Status](https://www.hl7.org/fhir/r4/nutritionorder-definitions.html#NutritionOrder.status)
 * [Intent](https://www.hl7.org/fhir/r4/nutritionorder-definitions.html#NutritionOrder.intent)
+* [Extension](http://hl7.org/fhir/R4/extension-event-basedon.html)
```

Comment:
I think the extension should be moved out from the list of supported fields and under it's own header of Extensions, similar to how it is done in the [patient resource](http://fhir.cerner.com/millennium/r4/individuals/patient/#extensions).

Reply:
[commit](https://github.com/cerner/fhir.cerner.com/pull/679/commits/e670be06e22931feb5f1e13e418287daff8befd7)
<img width="1296" alt="Screen Shot 2021-06-24 at 1 57 00 PM" src="https://user-images.githubusercontent.com/86370504/123318145-83023200-d4f4-11eb-84f4-2676954b938c.png">
<img width="960" alt="Screen Shot 2021-06-24 at 1 57 51 PM" src="https://user-images.githubusercontent.com/86370504/123318150-84335f00-d4f4-11eb-883a-1f90b7fae641.png">